misc/atrium-specs/build.gradle.kts

```diff
@@ -24,6 +24,7 @@ kotlin {
                 apiWithExclude("io.kotest:kotest-runner-junit5:$kotestVersion")
                 // necessary in order that intellij sees the io.kotest symbols (runner-junit5 actually already depends on it)
                 apiWithExclude("io.kotest:kotest-framework-api:$kotestVersion")
+                implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core-jvm:1.3.9")
```

Comment:
I'm sure if this should be done this way in Atrium's dependencies setup. Feel free to suggest better approach.

Without adding coroutines explicitly as implementation here, gradle is not able to compile the code.

Reply:
All fine, I can still improve after merge